### PR TITLE
random enemy

### PR DESCRIPTION
**Pull request by 19thanks96**
our enemy have random position x and y

### CODE REVIEW

**Comment by DarthVanger** [comment]
Please provide description for this pull request.

**Comment by DarthVanger** [comment]
This pull request has conflicts with fresh `master` branch.

### Please merge `master` into your branch

1. Switch to `master` with `git checkout master`.
2. Pull master with `git pull master`.
3. Switch to your branch with `git checkout randomenemy`.
4. Merge `master` into your branch with `git merge master`. It will show conflicts - fix them.
4.1. Use `git add` to mark resolved files.
4.2. Use `git commit` to finish the merge.
4.3. Use `git push` to push the merged branch with conflicts fixed.
5. Refresh pull request tab on github.com, check if it doesn't show conflicts anymore.
6. ✅ 

image

The conflicts happened because Dimon's Pull Request was merged.

**Review by DarthVanger — CHANGES_REQUESTED** [review]
Fix merge conflicts plz

File: app.js [hunk 1]
```diff
@@ -302,6 +303,8 @@ function moveShipLeft() {
 
 function moveShipRight() {
   airfighter.x += 10;
+  if (airfighter.x > airfighter.width) {
```
Comment (Alianshark) [inline app.js:306]: what is that doing?

**Review by 19thanks96 — COMMENTED** [review]
LGTM

**Review by Alianshark — APPROVED** [review]
LGTM

**Review by Gehinom — APPROVED** [review]

**Review by DarthVanger — APPROVED** [review]

**Pull request merged**